### PR TITLE
[CVP-1615]-adding new step for scorecard test

### PR DESCRIPTION
**Pull request by jsztuka**
The operator-sdk scorecard will need to be run on all operator bundle images that are tested with the Operator verification pipeline.
To this end, a new step implementing this will need to be added to the cvp-common workflow in the OpenShift-CI/DPTP Prow configuration.

### CODE REVIEW

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml
Comment (dirgim) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml:3]: I'm guessing we want to have the `cvp-operator-scorecard` here.

Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml:3]: Yes, forgot about this one. Thanks for noticing.

**Comment by jsztuka** [comment]
This PR can be merged only after : https://github.com/openshift/release/pull/16479 gets merged.

**Comment by jsztuka** [comment]
https://github.com/redhat-operator-ecosystem/release/pull/54 has been closed since the approach with literal image reference was used.

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml
Comment (petr-muller) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml:13]: IIRC we [changed this](https://github.com/openshift/release/pull/16489) in the other step to use the standard `test-credentials` namespace where the secret is automatically synced across clusters (more standard config -> fewer suprises )

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.metadata.json
Comment (petr-muller) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.metadata.json:6]: hrm, I guess we need to fix the parent OWNERS to better refect the reality (don't bother in this PR, I'll try to address this later)

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml
Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard-test/optional-operators-cvp-scorecard-test-ref.yaml:13]: Thanks for noticing, gonna change that.

File: ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml
Comment (dirgim) [inline ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml:13]: We need to run this test before the `optional-operators-cvp-common-test` because that one runs additional checks after all of the testing is done.

Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/aws/optional-operators-cvp-common-aws-workflow.yaml:13]: Changing the order.

File: ci-operator/step-registry/optional-operators/cvp-common/scorecard/OWNERS
Comment (petr-muller) [inline ci-operator/step-registry/optional-operators/cvp-common/scorecard/OWNERS:3]: I don't want nor need to be here.

Can this have 2 approvers from CVP and possibly more CVP people in `revievers`, too?

Comment (jsztuka) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/OWNERS:3]: Alright, I will add some of my team members here if thats ok.

Comment (dirgim) [reply ci-operator/step-registry/optional-operators/cvp-common/scorecard/OWNERS:3]: @petr-muller we'd still like to have members of DPTP in the list of `reviewers` at least until the rehearsal jobs are set up.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**
